mods/walls/init.lua

```diff
@@ -45,6 +45,12 @@ walls.register = function(wall_name, wall_desc, wall_texture_table, wall_mat, wa
 	})
 
 	-- crafting recipe
+	-- HACK:
```

Comment:
FYI. It's not a hack. It's a node registration convention enforced by `minetest.register_item`, so this change is totally reasonable.

Reply:
It is inconsistent that `minetest.register_craft` does not enforce the same convention, thus this is a workaround / "hack".